src/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 pybind11_add_module(_dupin dupininterface.cpp
-    dupin.h dupin.cpp
+    dupin.h dupin.cpp 
```

Comment:
Remove trailing whitespace. Also install pre-commit hooks to automatically do some of the formatting for you.

Reply:
added precommit hooks